pkg/test/certificates/foo.key

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
It seems that the certifcicate generation is an important part of these tests. I was wondering wether it wound't be better to add some functions for [generating certificates](https://golang.org/src/crypto/tls/generate_cert.go) dynamically (so we could skip all these fixtures in different formats)...

Reply:
That could be nice enhancement.  But I think it is not critical now.

Reply:
In a second thought, the idea of having the certificates as  fixtures is to ensure that the same certificate is retrieved each time. I'm thinking in the situation when we move some common setup for tests to a setup function that is reused in multiple tests.If in the test I need to check the same secret, how to ensure this if it is generated automatically?